src/wasm-exts/utf8.ml

```diff
@@ -0,0 +1,45 @@
+type codepoint = int
```

Comment:
maybe we can remove the other Utf8 decoder and use this only? Ditto the other `.mli` file.

Reply:
IIRC correctly, this version isn't tail-recursive and can blow up - is the other one written to be tail-recursive perhaps?

Reply:
Turns out, this file is not needed, `encoder.ml` references it as `Wasm.utf8`. But it uses it for (short?) identifiers only.